Hackerrank_problems/Extra Long Factorials/solution.py

```diff
@@ -0,0 +1,13 @@
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the extraLongFactorials function below.
+def extraLongFactorials(n):     
+
```

Comment:
This seems to be incomplete. Could you please check

Reply:
added the new solution in C .Please check it. @Vishruth-S